etc/systemd/system/zfs-volume@.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=ZFS zvol %I ready
+Before=zfs-volumes.target
+DefaultDependencies=no
+
+[Service]
+ExecStart=/bin/sh -c 'timeout=30 ; while [ $timeout -gt 0 ] ; do [ -e "/dev/zvol/$0" ] && exit 0 ; sleep 1 ; timeout=$((timeout-1)); done ; echo "zvol $0 unavailable"' %I
+Type=oneshot
+RemainAfterExit=true
```

Comment:
don't mix true/false/yes/no; systemd convention is yes/no

Reply:
Thanks, fixed this.